core/client/hdfs/src/main/java/alluxio/hadoop/AlluxioFileStatus.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.hadoop;
+
+import alluxio.client.file.URIStatus;
+
+import org.apache.hadoop.fs.FileStatus;
+import org.apache.hadoop.fs.Path;
+import org.apache.hadoop.fs.permission.FsPermission;
+
+/**
+ * Alluxio file status.
+ */
+public class AlluxioFileStatus extends FileStatus {
```

Comment:
what is the purpose of having this? seems like the two methods you overwrote did nothing but calling the super methods. 
If you want to DRY, you can create a helper function to convert a URIStatus + Path to a FileStatus object. This way you can get rid of this class.

Reply:
got it can yo remove the equals and hashCode methods from this class as they override nothing?

Reply:
Can not past findbugs after removing `hashcode` and `equals` method, so I have to keep those methods.